.github/workflows/ci.yml

```diff
@@ -226,8 +226,14 @@ jobs:
     timeout-minutes: 25
     strategy:
       matrix:
-        snuba_settings:
-          ["test", "test_distributed", "test_distributed_migrations"]
+        snuba_settings: ["test"]
+        clickhouse_image: ['yandex/clickhouse-server:20.3.9.70',
+                           'altinity/clickhouse-server:21.8.13.1.altinitystable']
+        include:
+          - snuba_settings: "test_distributed"
+            clickhouse_image: 'yandex/clickhouse-server:20.3.9.70'
+          - snuba_settings: "test_distributed_migrations"
```

Comment:
Why are these tests not done on 21.8?

Reply:
Running the distributed tests in parallel raises problems which I would like to punt fixing in the future. Here was a run where distributed tests were run in parallel https://github.com/getsentry/snuba/actions/runs/3147126533/jobs/5116339287

I would like to have something working and then iterating from there.

Reply:
OK makes sense.